master/reference/advanced/etcd-rbac/users-and-roles.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Creating Users and Roles in etcd
+---
+
+## Overview
+
+RBAC is *Role* Based Access Control so roles must be setup and also users
+that are given those roles must be created.  This document helps with the
+creation of both of those.
+
+> **Note:** The etcd release 3.x+ supports both v2 and v3 of its API.
+The etcd server keeps the roles and users separate, this means that if a
+user/role is created with the v2 API it will not appear in the v3 API.
+When adding roles and users they must be added through the API version that
+matches the version the component will be using.  This concern can be ignored
+if all roles and users are added through both API versions.
+
+## Users and Roles creation guides
+
+Use the following guides to setup your users, roles, and assignment of roles
```

Comment:
Yes - you are quite correct, it can't a pre-req because this doc tells you how to create the things!  D'oh.

leave as is.